Index.HTML

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
+    <title>YouTube</title>
+</head>
+<body>
+    <nav class="nav-bar">
+        <div id="logo-div">
+            <img src="https://archive.fo/Bss88/51a3832e6af6792add876216da444db23a4bea96.png" id="logo" alt="YouTube Logo">
+        </div>
+        <form>
+            <div id="search-div">
+                <input type="text" id="search-bar" placeholder="  Search">
+            </div>
+            <button type="submit" id="search-button">
+                <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" xmlns="http://www.w3.org/2000/svg">
+                    <path 
+                    d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
+                    fill="rgb(70,70,70)">
+                    </path>
+                </svg>
+            </button>
+        </form>
+        <div id="notifications">
+            <button type="submit" id="message-button" class="borderless">
+                <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M12.04 4C17 4 21 7.7 21 12.3c0 4.57-4 8.28-8.96 8.28-1.76 0-3.4-.47-4.8-1.28-.15-.1-.35-.1-.52-.04-1.58.68-2.8.88-3.45.93-.2 0-.34-.26-.22-.4.6-.9 1.08-1.8 1.43-2.5.1-.2.1-.46-.04-.65C3.58 15.4 3.1 13.9 3.1 12.3c0-4.6 4-8.3 8.94-8.3zm1.6 9.56v1.58c0 .17.22.25.34.13l3.43-3.34c.1-.07.1-.2 0-.27L14 8.32c-.12-.12-.33-.04-.33.13V10c0 .1-.07.18-.17.2-3.38.52-5.04 2.73-5.74 5.1-.06.2.2.34.32.2 1.3-1.42 3.05-2.1 5.4-2.13.1 0 .2.1.2.2z"
+                    fill="rgb(70,70,70)">
+                    </path>
+                </svg>
+            </button>
+            <button type="submit" id="notifications-button" class="borderless" >
+                <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M12 22c1.1 0 2-.9 2-2h-4c0 1.1.9 2 2 2zm6-6v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.67-1.5-1.5-1.5s-1.5.67-1.5 1.5v.68C7.63 5.36 6 7.92 6 11v5l-2 2v1h16v-1l-2-2z"
+                    fill="rgb(70,70,70)">
+                    </path>
+                </svg>
+            </button>
+            <div id="avatar-div">
+                <img src="/Icons/Captura.PNG" id="user-avatar" alt="User Avatar">
+            </div>
+        </div>
```

Comment:
The user avatar image is broken on live. We can fix it by removing the `/` to be `Icons/Captura.PNG`. [See this for more info](http://www.scriptingok.com/tutorial/HTML-links-2)